### PR TITLE
[Maistra-815][MAISTRA-633] Remove single tenant mode

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: topics/docs/getting_started/install.adoc [hunk 1]
```diff
@@ -76,62 +76,43 @@ NAME                              READY     STATUS    RESTARTS   AGE
 istio-operator-5cd6bcf645-fvb57   1/1       Running   0          1h
 ```
 
-=== Deploying the Istio Control Plane
+== Deploying the Istio Control Plane
+Maistra takes a multi-tenant approach, meaning that multiple control planes can be deployed in a cluster.
+To deploy a control plane, create a *ServiceMeshMemberRoll*, which lists the projects belonging to the
```
Comment (rcernich) [inline topics/docs/getting_started/install.adoc]: Do we need to document that the SMMR must be named `default` and must be present in the SCMP namespace?

Comment (brian-avery) [reply topics/docs/getting_started/install.adoc]: Is what's in the section on creating a ServiceMeshMemberRoll below enough?

Comment (JStickler) [reply topics/docs/getting_started/install.adoc]: If changing it will break it, yes we need to document it.  Customers will try changing things.

Comment (rcernich) [reply topics/docs/getting_started/install.adoc]: It's a little confusing, as you're telling the user partly what to do, but not exactly how to do it.  Maybe change the wording a bit to describe that a control plane is governed by two resources, smcp, which is used to configure the features of the control plane, and smmr, which is used to configure which projects are configured to work with the control plane.

Comment (brian-avery) [reply topics/docs/getting_started/install.adoc]: @rcernich  Thanks. Will change the wording. 

@JStickler this is an introduction stating what resources are required and describing what those two resources are. The two subsections of this delve into detail of creation, naming etc for the required resources. So not really a case of missing content. Will reword.

**Review by jwendell — APPROVED** [review]

**Review by rcernich — APPROVED** [review]

**Pull request merged**